packages/@sanity/default-login/README.md

```diff
@@ -19,14 +54,14 @@ static propTypes = {
   ]).isRequired,
   title: PropTypes.string,
   description: PropTypes.string,
-  sanityLogo: PropTypes.node
+  SanityLogo: PropTypes.component
```

Comment:
Don't think `PropTypes.component` exists? At least it's undocumented: https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
Doh, invalid readme. The code is ok though.